pyccl/tests/test_cosmology.py

```diff
@@ -10,6 +10,40 @@
 import pyccl as ccl
 
 
+def test_cosmo_methods():
+    """ Check that all pyccl functions that take cosmo
+    as their first argument are methods of the Cosmology object.
+    """
+    from inspect import getmembers, isfunction, signature
+    from pyccl import background, bcm, \
```

Comment:
I am not testing for all the modules here. Commit incoming...

Reply:
Done. I am now testing the `boltzmann` module as well.
Other than that, I added `neutrinos` and `tk3d` even though they contain no functions with `cosmo`, just in case there are any future additions in there (so that this part of the code won't need maintenance).
@damonge do you still approve?

Reply:
looks good to me, yes.